game/src/main/java/game/GameObjectInitializer.java

```diff
@@ -30,4 +30,10 @@ void createBonuses(ObjectManager<Bonus> bonusObjectManager) {
         bonusObjectManager.addObject(
                 gameObjectFactory.createBonus(new Vector(5, 5), new Vector(6, 6)));
     }
+
+    Tank createPlayer(ObjectManager<Tank> tankObjectManager) {
```

Comment:
method is named createPlayer but returns Tank :)

Reply:
Looks like its time to add Player entity which holds Tank as field. Player plays with this tank